src/transaction.rs

```diff
@@ -48,17 +51,17 @@ pub struct FundingTransaction {
 
 impl FundingTransaction {
     pub fn new(
-        (X_a, tid_a, amount_a): (OwnershipPublicKey, PartiallySignedTransaction, Amount),
-        (X_b, tid_b, amount_b): (OwnershipPublicKey, PartiallySignedTransaction, Amount),
+        (X_0, tid_0, amount_0): (OwnershipPublicKey, PartiallySignedTransaction, Amount),
```

Comment:
What's your reasoning behind `X_0` vs `X_a`? 
While it should not matter which is which it might be confusing if my `X_0` is different to your `X_0`. Should we go instead for `X_self` and `X_other`?

Reply:
> Should we go instead for X_self and X_other?

I specifically wanted to avoid using `X_self` and `X_other` because it implies that the order when calling the function matters.

> What's your reasoning behind X_0 vs X_a?

I was thinking that the `_a` and `_b` subscripts could imply Alice and Bob, whereas `_0` and `_1` more clearly convey "this is the first/second argument that you passed to this function, whatever it is".

It's all a little subjective though.

Reply:
I should make this an array like I did with all the other functions though.

Reply:
> I should make this an array like I did with all the other functions though.

... And then just sort it explicitly again.👍.